modules/apps/portal-store/portal-store-gcs/src/main/java/com/liferay/portal/store/gcs/GCSStore.java

```diff
@@ -315,6 +315,27 @@ private String _getFileVersionKey(
 			fileName, StringPool.SLASH, versionLabel);
 	}
 
+	private GoogleCredentials _getGoogleCredentials() throws IOException {
+		String serviceAccountKey = _gcsStoreConfiguration.serviceAccountKey();
+
+		if (Validator.isBlank(serviceAccountKey)) {
+			if (_log.isInfoEnabled()) {
+				_log.info(
+					"No credentials set for GCS Store. Library will default " +
+						"to using Application Default Credentials or " +
```

Comment:
I think we can leave out "or workload identity for auth" since this is a specific use of application credentials and we can be more general than that.

Reply:
Done! https://github.com/briangreenwald/liferay-portal/pull/29/commits/a2d702a13c49c524430b726c10343eb7a2e74ba2 (I actually cut even a bit more from the message, let me know of any problems.